prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/clinics/PM-OO/mailTemplates/InitialInvitation/bodyTemplate.html

```diff
@@ -41,10 +41,10 @@
       and Health Care Leaders to improve quality.
     </p>
     <p>
-      We have done everything we can to make sure that this survey does not go to families
-      who have lost a loved one. If you received this survey after your loved one has passed away,
-      please accept our heartfelt condolences and our sincere apology.
-      You may respond to this survey on behalf of your loved one if you would like.
+      We have done our best to ensure this survey does not reach families
+      who have lost a loved one. If you received it after a loss,
+      please accept our sincere condolences and apologies.
+      You may complete the survey on behalf of your loved one if you wish.
```

Comment:
This line is different from what is in other mails - is that intentional?
The others are
> You are welcome to complete the survey on their behalf if you wish.

Reply:
Right, it should be like the others
```suggestion
      You are welcome to complete the survey on their behalf if you wish.
```

Reply:
Amended the commit labeled `CARDS-2728: Update PM-OO email templates` for a cleaner history.